index.js

```diff
@@ -2900,6 +2947,14 @@
 
   return S;
 
+  };
+
+  //  Export two versions of the Sanctuary module: one with type checking;
+  //  one without.
+  var S       = createSanctuary(true);
+  S.unchecked = createSanctuary(false);
+  return S;
+
```

Comment:
I have many conflicting thoughts about it.

I like the API because

- It's easy to consume:
  * `var S = require('sanctuary').unchecked`
  * `import {unchecked as S} from 'sanctuary'`
- It's backwards compatible

I dislike it because:

- Consumption will become weird and fiddly when doing "unchecked in production":
  * `var S = isProduction ? require('sanctuary').unchecked : require('sanctuary')`
  * `import sanctuary from 'sanctuary'; const S = isProduction ? sanctuary.unchecked : sanctuary`
- Somewhere, knowing that there's an unnecessary sanctuary living in memory alongside the one I use doesn't sit well with me.
- It's not "future proofed" against additional options.

After spending some thought on it, a possible alternative could be:

```js
var creator = memoize(createSanctuary);
creator.create = createSanctuary;
return export;
```

to be consumed like:

```js
var S = require('sanctuary')(isProduction);
//or
import Sanctuary from 'sanctuary';
const S = Sanctuary(isProduction);
```

Or if you *must* create new instances:

```js
var S = require('sanctuary').create(isProduction);
```

Advantages of this approach:

- It's more recognizable: You're just instantiating the Sanctuary class, like with many other Node API's
- It's very easy to do "unchecked in production".
- It allows adding more options in the future
- You only need one instance in memory

Disadvantages are:

- It's horribly backwards incompatible